dev/NavigationView/NavigationViewItemPresenter.cpp

```diff
@@ -54,4 +64,23 @@ NavigationViewItem* NavigationViewItemPresenter::GetNavigationViewItem()
         navigationViewItem = winrt::get_self<NavigationViewItem>(item);
     }
     return navigationViewItem;
-}
\ No newline at end of file
+}
+
+void NavigationViewItemPresenter::UpdateCompactPaneLength(double compactPaneLength, bool shouldUpdate)
+{
+    if (!this) {
+        // "We" are null, skip
+        return;
+    }
```

Comment:
how can we be in this call if we are null?

Reply:
For some weird reason, during the lifecycle, there can be a state where accessing simple members of the NavigationViewItemPresenter fails. The specific scenario that failed for me, was accessing m_compacPaneLengthValue, where the member was simply not defined when in that method, resulting in a crash.

Reply:
strange.. I'd like to understand that better, can you test the crashing code path with your header file looking like this:

`double m_compactPaneLengthValue{40};`

Reply:
That didn't help either. Here a screenshot of live debugging:

![image](https://user-images.githubusercontent.com/16122379/75826273-714cc680-5da7-11ea-9458-4d02272346f3.png)

For some obscure reason, `this` is null ...